MultiTarget/EnabledTargetFrameworks.props

```diff
@@ -2,10 +2,14 @@
     <PropertyGroup>
         <UwpTargetFramework>uap10.0.17763</UwpTargetFramework>
         <WinAppSdkTargetFramework>net8.0-windows10.0.22621.0;net7.0-windows10.0.22621.0;net6.0-windows10.0.22621.0;</WinAppSdkTargetFramework>
-
+        
         <WasmHeadTargetFramework>net8.0</WasmHeadTargetFramework>
         <LinuxHeadTargetFramework>net8.0</LinuxHeadTargetFramework>
         <WpfHeadTargetFramework>net8.0</WpfHeadTargetFramework>
+        
```

Comment:
Nit: should likely remove these blank lines and the extra space on the line above

Reply:
they get added by the script anyway and these lines are left behind when you remove TFMs, we don't usually commit changes done by our tooling scripts but our default wasn't aligned with what the script was doing. Keeping this.